### PR TITLE
Installed project dependencies and set up webpack config file.

### PR DESCRIPTION
**Pull request by rick-o-H**

### CODE REVIEW

File: webpack.config.js [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+    rules: [
+      {
+        test: /\.(js|jsx)$/,
+        use: 'babel-loader',
+        exclude: /node_modules/
+      },
+      {
+        test: /\.css$/,
+        use: [
+          'style-loader',
```
Comment (Gledinburgh) [inline webpack.config.js:22]: I notice that style-loader, css-loader and url-loader are all here in the module object, but not installed as dependencies.  Is that typical?

File: webpack.config.js [hunk 2]
```diff
@@ -0,0 +1,50 @@
+const webpack = require('webpack');
```
Comment (Gledinburgh) [inline webpack.config.js:1]: webpack is imported here, but I don't see it used explicitly.

**Review by Gledinburgh — COMMENTED** [review]
This looks amazing! I got somewhat familiar with webpack throughout the day, but I have a few questions about a few of your lines. I left them as comments on the pull request.

File: webpack.config.js (hunk 1 above)
Comment (rick-o-H) [reply webpack.config.js:22]: @Gledinburgh Nice catch! CSS loader and URL loader were not whitelisted dependencies, so I was unsure about leaving that there for now and getting approval to use those packages( I believe we will have to if we are using any kind of custom CSS with webpack). But you are right, it's not typical to have loaders in the webpack.config file that you don't even have installed as dependencies.

File: webpack.config.js (hunk 2 above)
Comment (rick-o-H) [reply webpack.config.js:1]: @Gledinburgh Nice catch again!! Im not exactly sure as to why that is the case. I am thinking that the config object that is exported from the file is used with "webpack" somewhere behind the scenes when the transpiling and bundling is taking place.

**Pull request merged**